IBPSA/Electrical/BaseClasses/WindTurbine/PartialWindTurbine.mo

```diff
@@ -170,10 +170,15 @@ Below and above these wind speeds, the generated power is zero.
 </html>", revisions="<html>
 <ul>
 <li>
+April 9, 2025, by Michael Wetter:<br/>
+Increased small number to extend table beyond cut-out wind speed.<br/>
+<a href=\"https://github.com/lbl-srg/modelica-IBPSA/issues/1996\">IBPSA, issue 1996</a>.
```

Comment:
The provided URL is not correct as it refers to the github page of LBNL and the wrong issue number (1966 instead of 1996). It should be:
`<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1996\">IBPSA, issue 1996</a>.`